packages/template-retail-react-app/app/components/pagination/index.jsx

```diff
@@ -52,7 +48,7 @@ const Pagination = (props) => {
             {/* Previous Button */}
             <Button
                 {...styles.button}
-                as={isServer ? Link : RouteLink}
```

Comment:
We have to keep consistency between server and client

Reply:
What was the original intent of `Link` here instead of `RouteLink`? Do we know?